brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/ScribeReceiver.java

```diff
@@ -81,4 +77,11 @@ public ResultCode Log(final List<LogEntry> messages) throws TException {
         return result;
     }
 
+    public void setDelayMs(int delayMs) {
```

Comment:
Revert I mean just go back to accessing the field (similar to how the code
was originally). Since it is a test class, this sort of defensive
programming seems distracting, IMHO.